src/QMCWaveFunctions/Fermion/DiracDeterminantWithBackflow.cpp

```diff
@@ -142,7 +142,7 @@ void DiracDeterminantWithBackflow::registerData(ParticleSet& P, WFBufferType& bu
   buf.add(FirstAddressOfG, LastAddressOfG);
   buf.add(FirstAddressOfFm, LastAddressOfFm);
   buf.add(psiMinv.first_address(), psiMinv.last_address());
-  buf.add(LogValue);
+  buf.add(log_value_);
 }
 
 DiracDeterminantWithBackflow::LogValueType DiracDeterminantWithBackflow::updateBuffer(ParticleSet& P,
```

Comment:
LogValueType remains.

Reply:
Yes I'm not changing the LogValueType's that don't come in from WaveFunctionTypes, that's a locked type gained through inheritance.